### ISSUE TITLE
Web panel is slower

### ISSUE THREAD

**Issue by masterjek** [UnamSanctam/UnamWebPanel#145]
Good evening Unam. Today I switched to version 1.6.0 and the web panel began to work noticeably slower, sometimes it freezes when switching pages, sometimes the site itself does not open at all. I have a couple of questions.
1. Could this be related to changes in the new version?
2. Maybe this is due to the fact that I completely deleted the database and now all the workers are being restored again?
The panel works on a dedicated server on XMAPP
This was not the case with version 1.5.0.

**Comment by UnamSanctam** [comment]
Hmm, well I did change the database settings but theoretically it should be faster, try testing it like it is right now for a while and then maybe try removing this like: https://github.com/UnamSanctam/UnamWebPanel/blob/4064da1e79277d4f9eeb0f6db1f302d3c544f0d7/UnamWebPanel/class/db.php#L17 and test again to see if there's any difference.

**Comment by masterjek** [comment]
Deleted the line checked, nothing has changed in the work. With version 1.5.0 near the database file, two files constantly appeared and quickly disappeared.
1. unamwebpanel.db-journal
2. I don’t remember the second one exactly, but it seems that he has a word in his name - socache
In version 1.6.0 only one file appears - unamwebpanel.db-journal

**Comment by UnamSanctam** [comment]
Yes it's because I changed the journaling mode (for it to be faster), try changing the database file to this:
[unamwebpanel.zip](https://github.com/UnamSanctam/UnamWebPanel/files/8826401/unamwebpanel.zip), if it works then the problem is just with the DB Browser for SQLite being garbage (didn't preserve the journaling mode).

**Comment by UnamSanctam** [comment]
No it was just that DB Browser for SQLite is garbage, it didn't preserve the correct journaling setting before I uploaded it (it is supposed to be set to WAL in new version but it sets it to something else without displaying it).

**Comment by UnamSanctam** [comment]
Hmm, don't think that should matter (due to how many was added), try switching between them and see if you can see a change (I replaced the current v1.6.0 release with the changed one just so you know, so don't try that as the old version).

**Comment by masterjek** [comment]
Thanks a lot for your help. I checked on the same period the restoration of workers on a database with which the web panel slows down and with a database with which everything works very quickly. Workers recover the same way. Thanks again for your help and quick fix.
